src/multiversioning.zig

```diff
@@ -1306,6 +1306,36 @@ pub const Multiversion = struct {
                 // Close the handle before trying to execute.
                 posix.close(self.target_fd);
 
+                const pipe_name: [*:0]const u16 =
+                    std.unicode.utf8ToUtf16LeStringLiteral("\\\\.\\pipe\\") ++ random_wstr();
```

Comment:
I've gotten so used to seeing `++` only in the context of fully comptime known values, that this made my stop and stare for a little!

Reply:
Yeah, it's excessively neat that `++` requires only `len` to be `comptime`!